tests/SchedulePolicy/SchedulePolicyOrchestratorTest.php

```diff
@@ -152,7 +153,7 @@ public function testSchedulePolicyCanSortTasksUsingFirstInLastOut(): void
         ], $schedulePolicyOrchestrator->sort('first_in_last_out', ['foo' => $secondTask, 'app' => $task]));
     }
 
-    public function testTasksCanBeSortTasksUsingIdle(): void
+    public function testSchedulePolicyCanBeSortTasksUsingIdle(): void
```

Comment:
It is the schedule policy or the tasks that are sorted?

Reply:
ho yes sorry, you are right 🤣

Reply:
I thought there was a typo because tasks that can sort tasks seemed inconsistent to me.
without the `be` like so would be better ? 
`testSchedulePolicyCanSortTasksUsingIdle`

Reply:
Yes 🙂